### PR TITLE
Do not transform '_' named prop dependencies when creating a JS expression

### PR DESCRIPTION
**Pull request by AndyLaiJS**
Evidence by the testcase:
image

### CODE REVIEW

File: src/VCEL.Core/Helper/Constants.cs [hunk 1]
```diff
@@ -3,5 +3,6 @@
     public static class Constants
     {
         public const string DefaultContext = "vcelContext";
+        public const string ViewClientContext = "viewclientContext";
```
Comment (aaronbloom) [inline src/VCEL.Core/Helper/Constants.cs:6]: `underscoreContext` or just `_`

VCEL does not know about ViewClient

Comment (AndyLaiJS) [reply src/VCEL.Core/Helper/Constants.cs]: renamed to `underscoreContext` because `_._.env.user` doesn't look as nice

Comment (AndyLaiJS) [reply src/VCEL.Core/Helper/Constants.cs]: Updated to not include any of them. So `_.env.user` will still be `_.env.user`

File: src/Tests/VCEL.Test/ToJSCodeTests.cs [hunk 2]
```diff
@@ -27,6 +27,8 @@ public ToJsCodeTests()
     [TestCase("(D > 500000 or D < -500000)", "((vcelContext.D > 500000) || (vcelContext.D < -500000))")]
     [TestCase("(D > 123456789)", "(vcelContext.D > 123456789)")]
     [TestCase("(D > 1234567890)", "(vcelContext.D > 1234567890)")]
+    [TestCase("(_.env.user == 'jdoe2')", "(_.env.user === 'jdoe2')")]
+    [TestCase("(env._.user == 'jdoe2')", "(vcelContext.env._.user === 'jdoe2')")]
```
Comment (WillLiu-Eclipse) [inline src/Tests/VCEL.Test/ToJSCodeTests.cs:31]: can we add more test case to other categories.
e.g. in expr, match expr, let expr etc...

Comment (AndyLaiJS) [reply src/Tests/VCEL.Test/ToJSCodeTests.cs:41]: Sure, added!

**Review by aaronbloom — APPROVED** [review]

**Pull request merged**